docs/src/main/asciidoc/cdi-reference.adoc

```diff
@@ -1098,6 +1098,186 @@ public class MyService {
 In the above example, any code calling the `doSomething()` method triggers interception - in this case, the method becomes transactional.
 This is regardless of whether the invocation originated directly from the `MyService` bean (such as `MyService#doSomethingElse`) or from some other bean.
 
+=== Intercepting Producer Methods and Synthetic Beans
+
+By default, interception is only supported for managed beans (also known as class-based beans).
+To support interception of producer methods and synthetic beans, the CDI specification includes an `InterceptionFactory`, which is a runtime oriented concept and therefore cannot be supported in Quarkus.
+
+Instead, Quarkus has its own API: `InterceptionProxy` and `@BindingsSource`.
```

Comment:
I know that `@BindingsSource` is described below but it would not hurt to add a line in this paragraph too ...or remove `@BindingsSource` from this paragraph completely.

Reply:
Done.